src/odemis/gui/cont/tabs.py

```diff
@@ -499,6 +501,25 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _on_overview_visible(self, val):
+        """
+        Apply the visibility status of overview streams in the Localisation tab to
+        the overview streams in the Chamber tab.
+        """
+        # All the overview streams
+        acquired_streams = set()
+        acquired_streams.update(self.tab_data_model.overviewStreams.value)
+        # Visible overview streams
+        streams = self.tab_data_model.views.value[0].stream_tree.streams
+        visible_ov_streams = {st.stream for st in streams}
```

Comment:
This works because all the overview images use Projections, for now (that's why you have to write `st.stream`). However, it would be safer to handle both Stream and Projection. The View actually already has a function for that. So you can replace these 2 lines by:
`visible_ov_streams = set(self.tab_data_model.views.value[0].getStreams())`